### PR TITLE
Phantom types: background-repeat

### PR DESCRIPTION
**Pull request by BrianHicks**
This adds `background-repeat`

Progress on my bits of #392

### CODE REVIEW

File: src/Css.elm [hunk 1]
```diff
@@ -4014,6 +4015,29 @@ backgroundRepeat (Value repeat) =
...
+backgroundRepeat2 :
+    Value
+        { repeat : Supported
+        , space : Supported
+        , round : Supported
+        , initial : Supported
+        , unset : Supported
+        }
+    -> Style
+backgroundRepeat2 (Value repeat) =
```
[the remark this answers is not in the document]
Comment (BrianHicks) [reply src/Css.elm]: haaaaaaaaaaaaa yeah

File: src/Css.elm [hunk 2]
```diff
@@ -3978,6 +3987,110 @@ backgroundPosition4 (Value horiz) (Value horizAmount) (Value vert) (Value vertAm
...
+
+{-| [Repeat a background](#backgroundRepeat).
+-}
+repeat : Value { provides | repeat : Supported }
+repeat =
+    Value "repeat"
+
+
+{-| [Repeat a background](#backgroundRepeat) side to side.
+-}
```
Comment (rtfeldman) [inline src/Css.elm]: On these `Value` docs, can we get links to https://developer.mozilla.org/en-US/docs/Web/CSS/background-repeat#Values and mention that this compiles to the `"repeat-x"` value in CSS? (Similar to [this](https://github.com/rtfeldman/elm-css/pull/408/files#diff-854293bbb03f363ae4c16745b0af2a75R1200), for example.)

Code examples would also be great, even if they're copy/pasted from the `background` docs. 😄

File: src/Css.elm [hunk 3]
```diff
@@ -3978,6 +3987,110 @@ backgroundPosition4 (Value horiz) (Value horizAmount) (Value vert) (Value vertAm
...
+
+
+{-| Sets [`background-repeat`](https://css-tricks.com/almanac/properties/b/background-repeat/) along the horizontal axis, then the vertical axis.
+
+    backgroundRepeat2 repeat space
+
+    backgroundRepeat2 space round
+
+If you only need to set one value for both, see
+[`backgroundRepeat`](#backgroundRepeat) instead.
```
Comment (rtfeldman) [inline src/Css.elm]: Love the explanation and linking between `backgroundRepeat` and `backgroundRepeat2`! 😍

**Comment by BrianHicks** [comment]
did!

**Comment by rtfeldman** [comment]
Wonderful! Thank you @BrianHicks! 😻

**Pull request merged**